src/common/containers/builds.js

```diff
@@ -18,9 +18,9 @@ import styles from './container.css';
 class Builds extends Component {
   fetchInterval = null
 
-  fetchData({ snapLink, repository }) {
-    if (snapLink) {
-      this.props.dispatch(fetchBuilds(repository.url, snapLink));
+  fetchData({ snap, repository }) {
```

Comment:
If you do, can we talk about the shape of the actions and reducers for these and the other fetch actions we have?